src/pybind/mgr/mgr_module.py

```diff
@@ -450,13 +451,12 @@ def get_active_uri(self):
         return self._ceph_get_active_uri()
 
     def get_localized_module_option(self, key, default=None):
-        r = self.get_module_option(self.get_mgr_id() + '/' + key)
-        if r is None:
-            r = self.get_module_option(key)
-
+        r = self._ceph_get_module_option(key, self.get_mgr_id())
         if r is None:
-            r = default
-        return r
+            final_key = key.split('/')[-1]
```

Comment:
I still don't get this one. What value are we expecting in `key` for its splitting to be required?

`key` should be a regular option name (`server_port`). The localization is performed within that method (`x/server_port`). And `MODULE_OPTION_DEFAULTS` is indexed by internal option names.

Reply:
Ah.. yeah, I think you're right again here.  Pretty sure this can be removed.